syntax = "proto3";

package controller;

message movement {
	int32 direction = 1;
}

message button {
	int32 button_id = 1;
	int32 value = 2;
}

message controller_msg {
	oneof msg {
		movement go = 1;
		button btn = 2;
	}
}